### PR TITLE
ADD min length for service and service path

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: docs/multitenancy.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ The following syntax rules apply to service:
...
 * Only alphanumeric characters and underscores (`_`) are allowed.
 * Maximum length is 50 characters.
+* Minimum length is 1 character.
 * If uppercase characters are used, they are converted to lowercase.
 
 The following syntax rules apply to subservice:
 
 * It must start with the `/` character.
 * Maximum length is 50 characters.
+* Minimum length is 1 character.
```
Comment (kzangeli) [inline docs/multitenancy.md]: Not sure this solves the problem for the broker.
Orion would need to talk about service path *components* and that the minimum size of a component (what comes between slashes) is 1.
Now, perhaps this detail belongs to a different doc ... ?

Comment (fgalan) [inline docs/multitenancy.md]: This subtle detail belong to Orion doc. From an IoTP point of view, multipath sub services doesn't exist, so the current definition is correct.

Comment (kzangeli) [inline docs/multitenancy.md]: ok, NTC then :-)

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**